doc/thing/supported-boards/knot-dk.rst

```diff
@@ -0,0 +1,140 @@
+
+.. figure:: ../../../_static/dk.png
+   :scale: 50 %
+   :alt: DK - nRF52840 (PCA10056)
+   :align: center
+
+   DK - nRF52840 (PCA10056)
+
+More information about the board can be found at the `nRF52840 DK website
+<https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK>`_
+and `Zephyr nRF52840-PCA10056 website
+<https://docs.zephyrproject.org/latest/boards/arm/nrf52840_pca10056/doc/index.html>`_.
+The `Nordic Semiconductor Infocenter
+<https://infocenter.nordicsemi.com/topic/ug_nrf52840_dk/UG/nrf52840_DK/intro.html>`_
+contains the processor's information and the datasheet.
+
+-------------------------------------------------------------------------------
+
+Supported Features
+------------------
+
+The DK hardware provides support for the following devices:
+
+   - ADC
+   - CLOCK
+   - FLASH
+   - GPIO
+   - I2C
+   - MPU
+   - NVIC
+   - PWM
+   - RTC
+   - Segger RTT
+   - SPI
+   - UART
+   - USB
+   - WDT
+
+-------------------------------------------------------------------------------
+
+Connections and IOs
+-------------------
+
+.. warning:: The reserved buttons and LEDs should not be controlled by the user.
+
+Buttons
+'''''''
+
++-------+------------+---------------+---------------+------------+
+| Part  | Name       | Description   | Pin           | Permission |
++=======+============+===============+===============+============+
+| SW1   | Button 1   | Factory Reset | P0.11 / P1.07 | Reserved   |
++-------+------------+---------------+---------------+------------+
+| SW2   | Button 2   | Application   | P0.12 / P1.07 | Available  |
++-------+------------+---------------+---------------+------------+
+| SW3   | Button 3   | Application   | P0.24         | Available  |
++-------+------------+---------------+---------------+------------+
+| SW4   | Button 4   | Application   | P0.25         | Available  |
++-------+------------+---------------+---------------+------------+
+| SW5   | Boot/Reset | Reboot        |               | Reserved   |
++-------+------------+---------------+---------------+------------+
+
+LEDs
+''''
+
++-------+-------------+-------+-------+---------------+------------+
+| Part  | Description | Color | Pin   | Solder bridge | Permission |
++=======+=============+=======+=======+===============+============+
+| LED 1 | Status      | Green | P0.13 | SB5           | Reserved   |
++-------+-------------+-------+-------+---------------+------------+
+| LED 2 | Application | Green | P0.14 | SB6           | Available  |
++-------+-------------+-------+-------+---------------+------------+
+| LED 3 | Application | Green | P0.15 | SB7           | Available  |
++-------+-------------+-------+-------+---------------+------------+
+| LED 4 | Application | Green | P0.16 | SB8           | Available  |
++-------+-------------+-------+-------+---------------+------------+
+
+Analog Input
+''''''''''''
+
++-------+----------------+-------+------------+
+| Name  | Arduino Naming | Pin   | Permission |
```

Comment:
Please, see [this](https://infocenter.nordicsemi.com/topic/ug_nrf52840_dk/UG/nrf52840_DK/hw_analog_pins.html) page